.travis.yml

```diff
@@ -2,6 +2,7 @@ language: python
 env:
   - PYTHON_VERSION=2.7
 before_install:
+  - redis-server --version
```

Comment:
@squirrelo you mention that it was only in that box. Is that because that box has an older version of redis? If so, this means that we can hit that box again and run into the same issue that in this PR. Is it possible to specify the redis version that we need to travis do not send the build to a box with an outdated version of redis?